File: 2.Arrays/scores/scores.c
#include<cs50.h>
#include<stdio.h>
//全局变量
 const  int N=3;


int main(void){

  int scores[N];
  int sum=0;
  for(int i=0;i<N;i++){
    scores[i]=get_int("Scores: ");
    sum+=scores[i];
  }
  printf("Average:,%f\n",sum/(float)N);
}

File: 6.Python/scores.py
# scores = [72,73,33]

# average= sum(scores) / len(scores)

# print(f"Average: {average}")


from cs50 import get_int

scores=[]
for i in range(3):
  score = get_int("Score: ")
  # scores.append(score)
  scores=scores + [score]

average= sum(scores) / len(scores)
print(f"Average: {average}")
